ethereum/circuits/lib/src/misc/arrays.nr

```diff
@@ -32,12 +32,7 @@ pub(crate) fn memcpy_up_to_length<TItem, SRC_LEN, DEST_LEN>(dest: &mut [TItem; D
     }
 }
 
-/// Fills destination array with content of source array starting from the starting position.
-///
-/// # Arguments
-/// * `dest` - Destination array
-/// * `src` - Source array
-/// * `offset` - Offset in source array 
+// Fills destination array with content of source array starting from the starting position.
```

Comment:
Let's somehow include the fact that the `offset` is of source array, not destination